granCurrent.tsv

```diff
@@ -0,0 +1,34 @@
+Package	Version
```

Comment:
Ok, will this repo then need to track the directories inside the ./GRAN folder?  That all gets created by the build script right now.